src/fswAlgorithms/effectorInterfaces/solarArrayReference/solarArrayReference.h

```diff
@@ -0,0 +1,62 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#ifndef _SOLAR_ARRAY_REFERENCE_
+#define _SOLAR_ARRAY_REFERENCE_
+
+#include <stdint.h>
+#include "architecture/utilities/bskLogging.h"
+#include "cMsgCInterface/NavAttMsg_C.h"
+#include "cMsgCInterface/AttRefMsg_C.h"
+#include "cMsgCInterface/HingedRigidBodyMsg_C.h"
+
+#define EPS 1e-6                    //!< accuracy to check if a variable is zero
+
+/*! @brief Top level structure for the sub-module routines. */
+typedef struct {
+
+    /* declare these user-defined quantities */
+    double a1_B[3];                 //!< solar array drive axis in body frame coordinates
+    double a2_B[3];                 //!< solar array surface normal at zero rotation
+    int bodyFrame;                  //!< flag = 1: compute theta reference based on current attitude instead of attitude reference
```

Comment:
:large_orange_diamond: This variable could help the code in the implementation be more clarifying by making it an enum. Something like (but you could choose good enum strings
```
enum BodyAttitudeFrame {
    REFERENCE_ATTITUDE = 0,
    CURRENT_ATTITUDE = 1
};
BodyAttitudeFrame bodyFrame; 
```

In general, if using an int as a flag, it's valuable to consider if an enum would make the implementation (.c/cpp) more clarifying/readable.

Reply:
The change was implemented as suggested.